htdocs/core/class/html.form.class.php

```diff
@@ -1563,10 +1563,21 @@ public function select_remises($selected, $htmlname, $filter, $socid, $maxvalue
 						$tmpfac = new Facture($this->db);
 						if ($tmpfac->fetch($obj->fk_facture_source) > 0) {
 							$desc = $desc.' - '.$tmpfac->ref;
+							$multicurrency_code = $tmpfac->multicurrency_code;
 						}
 					}
+					// show Source Supplier Invoice Ref
+					else {
+        				if (!empty($conf->global->MAIN_SHOW_FACNUMBER_IN_DISCOUNT_LIST) && !empty($obj->fk_invoice_supplier_source)) {
+							$tmpfac = new FactureFournisseur($this->db);
```

Comment:
Tabulation must use tab, not spaces.

Reply:
Thanks. not a programmer, I am trying my best to understand the dolibarr code and tweak it to express my humble suggestions.